instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/db/DbRequestDecorator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2.db;
+
+import io.opentelemetry.instrumentation.awssdk.v2_2.SdkRequestDecorator;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.attributes.SemanticAttributes;
+import software.amazon.awssdk.core.SdkRequest;
+import software.amazon.awssdk.core.interceptor.ExecutionAttributes;
+import software.amazon.awssdk.core.interceptor.SdkExecutionAttribute;
+
+public final class DbRequestDecorator implements SdkRequestDecorator {
+
+  @Override
+  public void decorate(Span span, SdkRequest sdkRequest, ExecutionAttributes attributes) {
+
+    span.setAttribute(SemanticAttributes.DB_SYSTEM.key(), "dynamodb");
```

Comment:
`SemanticAttributes.DB_SYSTEM.set(span, "dynamodb")` makes it easier to find non-semantic attributes in the future